specification/securityinsights/data-plane/Microsoft.SecurityInsights/preview/2022-12-01-preview/ThreatIntelligence.json

```diff
@@ -0,0 +1,187 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Sentinel Upload Indicators API",
+    "description": "Spec for Microsoft Sentinel Upload Indicators API.",
+    "version": "2022-12-01-preview"
+  },
+  "host": "sentinelus.azure-api.net",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/workspaces/{workspaceId}/threat-intelligence/indicators:upload": {
+      "post": {
+        "x-ms-examples": {
+          "Upload Threat Intelligence indicators": {
+            "$ref": "./examples/threatintelligence/UploadThreatIntelligence.json"
+          }
+        },
+        "summary": "Upload the indicators to the workspace",
+        "description": "Upload the list of indicators to the workspace specified",
+        "operationId": "threat-intelligence/indicators:upload",
```

Comment:
This doesn't look right. operationIds are generally "<operationGroup>_<operationName>" .. no slashes or colons.

Reply:
we will replace the "/" with the "_"

Reply:
We will change this to "ThreatIntelligenceIndicators_Upload"

Reply:
Autorest requires "indicators:upload" to be part of the operationId so, we're changing it to ThreatIntelligence_indicators:upload. Is that okay?

Reply:
What message are you getting from Autorest? I don't think I've seen this issue before.

Reply:
This is the warning I get: 
warning | PostOperationIdContainsUrlVerb | OperationId should contain the verb: 'indicators:upload' in:'ThreatIntelligenceIndicators_Upload'. Consider updating the operationId